<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_guiClass</class>
 <widget class="QMainWindow" name="qt_guiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>898</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt_gui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="verticalWidget_3" native="true">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="2">
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="0" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="verticalWidget_2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QSplitter" name="splitter_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <widget class="QWidget" name="">
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QGroupBox" name="groupBox_4">
                    <property name="title">
                     <string>Image List</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_9">
                     <item row="1" column="0">
                      <widget class="QListWidget" name="listWidget"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QGroupBox" name="groupBox">
                 <property name="title">
                  <string>Task Properties</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Task Type</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="comboBox">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="editable">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Implementation Type</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QComboBox" name="comboBox_2">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>16777215</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="0">
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>Repeats Count</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="1">
                   <widget class="QSpinBox" name="spinBox">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>999999</number>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Blurring Kernel Size</string>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="1">
                   <widget class="QSpinBox" name="spinBox_2">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="layoutDirection">
                     <enum>Qt::LeftToRight</enum>
                    </property>
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>999</number>
                    </property>
                    <property name="singleStep">
                     <number>2</number>
                    </property>
                   </widget>
                  </item>
                  <item row="6" column="0">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Sigma X</string>
                    </property>
                   </widget>
                  </item>
                  <item row="8" column="0">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>Output Directory</string>
                    </property>
                   </widget>
                  </item>
                  <item row="8" column="1">
                   <widget class="QLineEdit" name="lineEdit">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="9" column="0">
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Modify Original</string>
                    </property>
                   </widget>
                  </item>
                  <item row="9" column="1">
                   <widget class="QComboBox" name="comboBox_3"/>
                  </item>
                  <item row="6" column="1">
                   <widget class="QDoubleSpinBox" name="doubleSpinBox">
                    <property name="value">
                     <double>1.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="10" column="0">
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Optimize Block Processing</string>
                    </property>
                   </widget>
                  </item>
                  <item row="10" column="1">
                   <widget class="QComboBox" name="comboBox_4"/>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>Processing</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_8">
                 <item row="0" column="0">
                  <widget class="QPushButton" name="pushButton_2">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="text">
                    <string>Add</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QPushButton" name="pushButton_3">
                   <property name="text">
                    <string>Remove</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QPushButton" name="pushButton_4">
                   <property name="text">
                    <string>Run Task</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QPushButton" name="pushButton">
                   <property name="text">
                    <string>Show</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Results</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0" rowspan="2">
            <widget class="QTreeWidget" name="treeWidget">
             <property name="columnCount">
              <number>2</number>
             </property>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string notr="true">2</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
